domains/thefthelper.is-local.org.json

```diff
@@ -0,0 +1,14 @@
+{
+    "domain": "is-local.org",
+    "subdomain": "thefthelper",
+
+    "owner": {
+        "email": "kibandhonor@gmail.com"
+    },
+
+    "record": {
+        "A": ["127.0.0.1"]
```

Comment:
Invalid A record, local loopback IP will not work.

Reply:
Oh, lemme change it later